app/proguard-rules/picasso-rules.pro

```diff
@@ -1,7 +1,2 @@
--keep class com.squareup.picasso.** { *; }
--keepclasseswithmembers class * {
-    @com.squareup.picasso.** *;
-}
--keepclassmembers class * {
-    @com.squareup.picasso.** *;
-}
+#Picasso
+-dontwarn com.squareup.okhttp.**
```

Comment:
Do you know what has changed in Picasso that the existing rules are not longer correct?